test/unit/resources/elasticsearch_test.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Rony Xavier, rx294@nyu.edu
+# license: All rights reserved
```

Comment:
Please remove this line. InSpec is released under Apache 2.0. We have a pending PR to remove this line from all our files (acknowledging that you probably copy/pasted this from another existing file 🙂 ).